src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPicked.cpp

```diff
@@ -261,11 +262,12 @@ namespace OpenMS
     }
 
     //---------------------------------------------------------------------------
-    //Step 1:
-    //Precalculate intensity scores for peaks
+    // Step 1:
+    // Precalculate intensity scores for peaks
     //---------------------------------------------------------------------------
-    if (debug_) log_ << "Precalculating intensity thresholds ..." << std::endl;
-    //new scope to make local variables disappear
+    if (debug_)
+      log_ << "Precalculating intensity thresholds ..." << std::endl;
```

Comment:
When using more than one line for an `if` statement, please use brackets, like in the previous one in line 256, even if only one line ends up in the scope of the statement. Or leave these as one-line statements as they were before.
It might contradict with some other style guides, but this is an OpenMS specific style choice.

Reply:
also applies to the multiple other cases below